### DISCUSSION TITLE
21-06-07 일일회고

### DISCUSSION THREAD

**Discussion by GleamingStar** [d-h-k/issue-tracker#3] [회고]
만나서 반갑습니다! 👋

**Comment by JacksonPk** [comment]
좋은 분들 만나서 좋았습니다~ 
앞으로 3주동안 잘 지내봐요. 아프지말고

**Comment by eeeesong** [comment]
다들 만나서 반갑습니다🤓
사실 저는 오늘 잠을 거의 못자서 굉장히 몽롱한 상태였는데
다들 진행을 척척 너무 잘해주셔서 굉장히 편안한 첫날이었던 것 같습니다!
마지막 프로젝트라 좋은 성과 내고 싶다는 마음도 크지만
다들 건강 지켜가면서 끝까지 재미있게 했으면 좋겠어요! 화이팅!!🔥

**Comment by GleamingStar** [comment]
다들 만나서 반갑습니다!!!!

뭔가 오늘 독단적으로 저 하고 싶은 대로 정해버린 것 같아서 나머지 분들께 죄송스럽네요 🤦‍♂️
지금도 안늦었어요! 이상한 부분, 불만인 부분, 왜 이렇게 하는지 이해가 안되는 부분 바로 말씀해주세요!!

카페에서 귀갱한 점도 다섯 분께 죄송합니다 ㅠㅠ 🙏
아마 프로젝트 내내 경험하실텐데... 오늘처럼 말 많이해야되는거 아니면 아마 채팅으로해서 괜찮을겁니다

3주동안 잘부탁드리고 내일 오전에 코쿼에서봬요!!

**Reply by GleamingStar** [reply]
제가 Seong입니다! 👋

**Comment by Jiwon-JJW** [comment]
저녁 느즈막히 올리는 회고네요 ㅎㅎ 
좋으신분들과 만나뵙게 되어 영광입니다!!
많이 부족한 부분이 있을 수 있겠지만, 최대한 열심히 피해안끼치도록 힘내보겠습니다 🥺👍

아프지 않고 코딩과 건강 모두 챙기는 3주가 되었으면 좋겠네요!!
화이팅!!

**Comment by reminderclock** [comment]
이번 프로젝트 무사히 유종의 미를 거두었으면 좋겠네요. 다들 의미있는 3주가 되시길 바라겠습니다 1팀 화이팅!
